apps/examples/testcase/ta_tc/media/itc/itc_media_fileinputdatasource.cpp

```diff
@@ -215,16 +223,14 @@ static void itc_media_FileInputDataSource_read_p(void)
 	memset(buf, 0, 31);
```

Comment:
Same, Dont hardcode, use #defines if possible.

Reply:
changes done and code also updated in gerrit